mage_ai/server/server.py

```diff
@@ -229,9 +235,21 @@ async def main(
                     email='admin@admin.com',
                     password_hash=create_bcrypt_hash('admin', password_salt),
                     password_salt=password_salt,
-                    owner=True,
+                    roles_new=[Role.query.filter(Role.name == 'owner').first()],
                     username='admin',
                 )
+        else:
+            if not user.roles_new:
```

Comment:
Is this a one-off migration from the old roles to the new roles? If so, I would suggest do a batch update here and commit at the end. Otherwise, there could be issues if the update fails in the middle.